docs: Updated documentation according with latest webui changes (#5579)
eclipse/kura: docs/connect-field-devices/asset-implemetation.md

The following **mode**s are implemented for computing scale and offset:

> 

Author: IMO, the term **mode** here is a bit misleading. Is it better to present her the **scaleoffset.type** instead?

Reviewer: with @nicolatimeus we decided to adopt this name, let see what he thinks about.

Example of `INTEGER` **mode**:
channel configured with:

- **value.type**: `INTEGER`
- **scaleoffset.type**: `DEFINED_BY_VALUE_TYPE`
- **scale**: 3
- **offset**: 1

Since **scaleoffset.type** is `DEFINED_BY_VALUE_TYPE`, the **mode** determined by the **value.type** parameter (`INTEGER`) will be used.

If `channel_value` is 5 (INTEGER) the result is: `(int) (channel_value * (int) 3 + (int) 1) = 16`.

Example of `DOUBLE` **mode**:
channel configured with:

- **value.type**: `DOUBLE`
- **scaleoffset.type**: `DEFINED_BY_VALUE_TYPE`
- **scale**: 3.25
- **offset**: 1.5

If `channel_value` is 5 (DOUBLE) the result is: `(double) (channel_value * (double) 3.25d + (double) 1.5d) = 17.75`.

> As the examples show, the final result can be different depending on the used **mode** and **value.type**.

Reviewer 2: Wording

```suggestion
As the examples show, the final result can change depending on the **mode** and **value.type** used.
```

### Replace current channels

> The `Replace current channels` must be checked when the user wants to replace **all** the channels currently present in the asset, with the ones that will be created by the csv uploading.

Reviewer 2: Is this more readable? Feel free to ignore this if not.

```suggestion
Checking `Replace current channels` when uploading a `csv` file will result in **all** the channels in the asset being replaced by the `csv` file content.
```
